js/map.js

```diff
@@ -54,27 +54,31 @@
       mapPin.hidden = true;
     });
   };
-  window.load(renderPins); // пока без второго параметра
+  window.load(renderPins); // загрузка данных
 
   /* АКТИВАЦИЯ ФОРМЫ */
   const mapBooking = document.querySelector(`.map`);
+
+  const deactive = () => {
```

Comment:
```suggestion
  const deactivate = () => {
```
А вобще лучше fadeMap